src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -98,13 +98,27 @@
         Lists.newArrayList(Collections2.transform(repairSegments, segment -> segment.getTokenRange())));
 
     String repairUnitClusterName = repairUnitOpt.get().getClusterName();
-    String metricNameForRepairProgress = metricName("repairProgress",  repairUnitClusterName, repairRunId);
+    String repairUnitKeyspaceName = repairUnitOpt.get().getKeyspaceName();
+    String metricNameForRepairProgress = metricName("repairProgress",
+                                                     repairUnitClusterName,
+                                                     repairUnitKeyspaceName,
+                                                     repairRunId);
     context.metricRegistry.register(metricNameForRepairProgress, (Gauge<Float>) ()  -> repairProgress);
-    String metricNameForSegmentsDone = metricName("segmentsDone",  repairUnitClusterName, repairRunId);
-    context.metricRegistry.register(metricNameForSegmentsDone, (Gauge<Float>) ()  -> segmentsDone);
-    String metricNameForSegmentsTotal = metricName("segmentsTotal",  repairUnitClusterName, repairRunId);
-    context.metricRegistry.register(metricNameForSegmentsTotal, (Gauge<Float>) ()  -> segmentsTotal);
-    metricNameForMillisSinceLastRepair = metricName("millisSinceLastRepair", repairUnitClusterName, repairRunId);
+    metricNameForMillisSinceLastRepair = metricName("millisSinceLastRepair",
+                                                     repairUnitClusterName,
+                                                     repairUnitKeyspaceName,
```

Comment:
I'm afraid that we may be changing the name of a metric that people might already be monitoring. Would you mind duplicating the metric so that both the new and the old one will be there ?